### PR TITLE
feat: add integrations page

### PR DESCRIPTION
**Pull request by lotyp**
## What was changed

Added a new "Integrations" section to the Cycle ORM documentation to showcase the available external integration packages that allow the use of Cycle ORM in other PHP frameworks.

## Why?

To inform users about available integrations with popular PHP frameworks, enhancing Cycle ORM's usability across different development environments. This update aims to support developers in leveraging Cycle ORM's features more effectively within their preferred frameworks.

## Checklist

- Any docs updates needed?
  This PR serves as the update; no further documentation changes are required.

### CODE REVIEW

**Comment by lotyp** [comment]
Probably, Spiral Framework should also be mentioned here? 😅

File: docs/en/manifest.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
       "contributing": "Contributing",
       "issue-test-case": "Issue with Test Case",
       "intro/semver": "Versioning",
-      "license": "LICENSE"
+      "license": "LICENSE",
+      "integrations": "Integrations"
```
Comment (msmakouz) [inline docs/en/manifest.json]: Perhaps it's worth adding this item to the menu above the 'License' item? @roxblnfk What do you think?

File: docs/en/integrations.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Integrations
+
+Cycle ORM is designed to be flexible and extensible, allowing it to integrate seamlessly with various PHP frameworks. This capability ensures developers can leverage Cycle ORM's powerful features across different environments, enhancing productivity and maintaining consistency in data management practices. Below are the currently available official and community-supported integration packages that facilitate the use of Cycle ORM within other PHP frameworks, including its official integration with the Spiral framework.
+
+## Available Integrations
+
+### Spiral Framework
+
+Cycle ORM serves as the official ORM for the Spiral framework, supported by the `spiral/cycle-bridge` package. This integration is optimized for Spiral applications, ensuring compatibility and performance, especially with the [Roadrunner](https://roadrunner.dev) server for long-running applications.
```
Comment (msmakouz) [inline docs/en/integrations.md]:
```suggestion
Cycle ORM serves as the official ORM for the Spiral framework, supported by the `spiral/cycle-bridge` package. This integration is optimized for Spiral applications, ensuring compatibility and performance, especially with the [RoadRunner](https://roadrunner.dev) server for long-running applications.
```

**Review by msmakouz — CHANGES_REQUESTED** [review]

File: docs/en/manifest.json (hunk 1 above)
Comment (lotyp) [reply docs/en/manifest.json]: Adjusted the doc structure to place "Integrations" right after "Installation". This mirrors the dev workflow, making it easier to jump from setup to exploring integrations.

**Review by msmakouz — APPROVED** [review]

File: docs/en/intro/integrations.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Spiral Framework
+
+Cycle ORM serves as the official ORM for the Spiral framework, supported by the `spiral/cycle-bridge` package. This integration is optimized for Spiral applications, ensuring compatibility and performance, especially with the [RoadRunner](https://roadrunner.dev) server for long-running applications.
+
+- **GitHub Repository**: [spiral/cycle-bridge](https://github.com/spiral/cycle-bridge)
+- **Documentation**: [Spiral Framework Integration](https://spiral.dev/docs/basics-orm)
+- **Maintained By**: [SpiralScout](https://spiralscout.com)
+
+### Yii Framework
```
Comment (roxblnfk) [inline docs/en/intro/integrations.md]:
```suggestion
## Yii 3
```

File: docs/en/intro/integrations.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The Yii Framework integration is officially supported through a package maintained by the [YiiSoft](https://www.yiiframework.com). This package ensures that Cycle ORM works seamlessly with Yii, providing a robust solution for managing database operations within Yii applications.
+
+- **GitHub Repository**: [yiisoft/yii-cycle](https://github.com/yiisoft/yii-cycle)
+- **Documentation**: [Guide and Usage](https://github.com/yiisoft/yii-cycle/tree/master/docs/guide/en)
+
+### Laravel Framework
```
Comment (roxblnfk) [inline docs/en/intro/integrations.md]:
```suggestion
## Laravel
```

File: docs/en/intro/integrations.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Integrations
+
+Cycle ORM is designed to be flexible and extensible, allowing it to integrate seamlessly with various PHP frameworks. This capability ensures developers can leverage Cycle ORM's powerful features across different environments, enhancing productivity and maintaining consistency in data management practices. Below are the currently available official and community-supported integration packages that facilitate the use of Cycle ORM within other PHP frameworks, including its official integration with the Spiral framework.
+
+## Available Integrations
+
+### Spiral Framework
```
Comment (roxblnfk) [inline docs/en/intro/integrations.md]:
```suggestion
## Spiral
```

File: docs/en/intro/integrations.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# Integrations
+
+Cycle ORM is designed to be flexible and extensible, allowing it to integrate seamlessly with various PHP frameworks. This capability ensures developers can leverage Cycle ORM's powerful features across different environments, enhancing productivity and maintaining consistency in data management practices. Below are the currently available official and community-supported integration packages that facilitate the use of Cycle ORM within other PHP frameworks, including its official integration with the Spiral framework.
+
+## Available Integrations
```
Comment (roxblnfk) [inline docs/en/intro/integrations.md]:
```suggestion
```

**Comment by roxblnfk** [comment]
https://discord.com/channels/538114875570913290/630705736019869719/1222536717517455472 might be added
https://github.com/technoly/neos-eventstore-cycleadapter

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**